js/main.js

```diff
@@ -1,2 +1,179 @@
 $(function () {
+  let choice = 'none'
+  let compHand = ''
+  let winner = ''
+  let score = 0
+
+  $('#rock').click(() => {
+    compHand = generateHand()
+    winner = evalMatch('rock',compHand)
+    updateScore(winner)
+  })
+
+  $('#paper').click(() => {
+    compHand = generateHand()
+    winner = evalMatch('paper',compHand)
+    updateScore(winner)
+  })
+
+  $('#scissors').click(() => {
+    compHand = generateHand()
+    winner = evalMatch('scissors',compHand)
+    updateScore(winner)
+  })
+  $('#restart').click(() => {
+    clearScore()
+    clearRounds()
+    location.reload()
+  })
+
+  function generateHand() {
+    let num = Math.floor(Math.random()*3)+1
+    if (num === 1){
+      return 'rock'
+    }
+    else if (num === 2){
+      return 'paper'
+    }
+    else if (num === 3){
+      return 'scissors'
+    }
+    else {
+      return 'error'
+    }
```

Comment:
This works perfectly well and I love that you're taking advantage of the random number built in method! [Here](https://github.com/jsd20200127/rock-paper-scissors-solution/blob/502d85718e17749df951e7808338ada1ec6d1844/js/main-with-bonus.js#L28-L32) is alternative sample solution that takes advantage of the same method, as well as indexes in arrays for reference.